README.md

```diff
@@ -20,7 +20,7 @@ Language server client for haskell using the [HIE](https://github.com/haskell/ha
 * Diagnostics via HLint and GHC warnings/errors
 * Code actions and quick-fixes via [`apply-refact`](https://github.com/mpickering/apply-refact) (click the lightbulb)
 * Type information and documentation (via hoogle) on hover
-* Jump to definition (`F12` or `Go to Definition` in command palette)
+* Jump to definition (`F12` or `Go to Definition` in command palette). Note that this only works for definitions within the current file.
```

Comment:
I think this limitation is gone now in newer versions of ghcide (and hls)

Reply:
But it still does not work for non local package definitions (i.e. libraries)